src/app/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -455,7 +454,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 17.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 15.0;
```

Comment:
So IPHONEOS_DEPLOYMENT_TARGET is for the minimum version?

Reply:
Yes it is according to this https://docs.flutter.dev/deployment/ios#review-xcode-project-settings

> iOS Deployment Target
>    The minimum iOS version that your app supports. Flutter supports iOS 12 and later. If your app or plugins include >Objective-C or Swift code that makes use of APIs newer than iOS 12, update this setting to the highest required version.